examples/bell_state_stabilization.md

```diff
@@ -0,0 +1,471 @@
+        generator_type=SGType.SHFSG,
+        analyzer_type=QAType.SHFQA,
+        pqsc_mode=PQSCMode.DECODER
+    )
+)
+
+# Calculate the QA delay
+int_len = shfqa.qachannels[QA_CHANNEL].readout.integration.length()
+int_delay = round(shfqa.qachannels[QA_CHANNEL].readout.integration.delay() * SAMPLE_RATE)
+qa_delay = int_len + int_delay
+
+# Get the total feedback latency from the model
+feedback_latency = feedback_model.get_latency(qa_delay)
+```
+
+### Define SeqC program and Command Table
+Finally, we configure the error correction pulses that the SHFSG should apply. We have to apply gates both to the data qubits (to correct the errors) and to the ancilla qubits (to reset them in the initial state), for a total of 4 qubits. Therefore, we use 4 AWG cores.
+
+```python
+data_A_awg = shfsg.sgchannels[0].awg
+data_B_awg = shfsg.sgchannels[1].awg
+ancilla_x_awg = shfsg.sgchannels[2].awg
+ancilla_z_awg = shfsg.sgchannels[3].awg
+```
+
+The output of the Decoder Unit sent over ZSync is always an 8-bit number, but the SHFSG will only need to look at the two least significant bits. Even more precisely, each AWG core will have to look at only one of these two bits: for the first data qubit and the Z ancilla only the second least significant bit is relevant, while for the second data qubit and the X ancilla only the least significant bit is relevant. (You can find very clear pictures in [this blog post](https://www.zhinst.com/ch/en/blogs/bell-state-stabilization-superconducting-qubits-real-time-feedback)).
+
+In the following lines we tell each AWG cores which bits to read. We can configure it by telling:
+- if and how to shift the bits
+- which bits to select after the shift: this is done by specifying a mask
+- if and how to apply ad additive offset to the bits (not needed here)
+
+```python
+with shfsg.set_transaction():
+    data_A_awg.zsync.decoder.shift(1)
+    data_A_awg.zsync.decoder.mask(0b1)
+    data_A_awg.zsync.decoder.offset(0)
+
+    data_B_awg.zsync.decoder.shift(0)
+    data_B_awg.zsync.decoder.mask(0b1)
+    data_B_awg.zsync.decoder.offset(0)
+
+    ancilla_x_awg.zsync.decoder.shift(1)
+    ancilla_x_awg.zsync.decoder.mask(0b1)
+    ancilla_x_awg.zsync.decoder.offset(0)
+
+    ancilla_z_awg.zsync.decoder.shift(0)
+    ancilla_z_awg.zsync.decoder.mask(0b1)
+    ancilla_z_awg.zsync.decoder.offset(0)
+
+
+```
+
+Lastly, we write the sequencer program and the command table for the SHFSG. With the following instruction:
+
+`executeTableEntry(ZSYNC_DATA_PQSC_DECODER, feedback_latency);`
+
+the Decoder data will be read after waiting for the feedback latency, and it will be directly forwarded to the command table without going through the sequencer. The command table will then execute the table entry corresponding to the bit read from the Decoder data. For illustrative purposes, to distinguish between X gate and Z gate we define two sequencer programs that differ only for the duration of the pulse. 
```

Comment:
```suggestion
The Decoder data will be read after waiting for the feedback latency, and it will be directly forwarded to the command table without going through the sequencer. The command table will then execute the table entry corresponding to the bit read from the Decoder data. For illustrative purposes, to distinguish between X gate and Z gate we define two sequencer programs that differ only for the duration of the pulse. 
```